src/coreclr/vm/threads.cpp

```diff
@@ -1128,6 +1127,7 @@ void InitThreadManager()
 
     }
     else
+#endif // !FEATURE_PORTABLE_HELPERS
```

Comment:
Nit: This #endif can be after the else part right before `#ifndef TARGET_UNIX`. Also, `JIT_WriteBarrier_Loc` can be under `#ifndef FEATURE_PORTABLE_HELPERS`

Reply:
I will fold this feedback into the next PR.